### PR TITLE
Add xyz.safeworlds.hiit

### PR DESCRIPTION
**Pull request by mfep**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flatpak.org/en/latest/first-build.html

### CODE REVIEW

File: xyz.safeworlds.hiit.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- --socket=pulseaudio
+- --device=dri
+- --env=RUST_LOG=gtk_rust_template=debug
+- --env=G_MESSAGES_DEBUG=none
+- --env=RUST_BACKTRACE=1
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm16/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm16/lib
+  build-args:
+  - --share=network
```
Comment (hfiguiere) [inline xyz.safeworlds.hiit.yml:22]: this is not allowed and the build will fail because of it.

File: xyz.safeworlds.hiit.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- org.freedesktop.Sdk.Extension.rust-stable # 23.08
+- org.freedesktop.Sdk.Extension.llvm16
+command: hiit
+finish-args:
+- --share=ipc
+- --socket=fallback-x11
+- --socket=wayland
+- --socket=pulseaudio
+- --device=dri
+- --env=RUST_LOG=gtk_rust_template=debug
```
Comment (hfiguiere) [inline xyz.safeworlds.hiit.yml:15]: why would you run the app with the verbose debug output?

File: xyz.safeworlds.hiit.yml (hunk 1 above)
Comment (mfep) [reply xyz.safeworlds.hiit.yml]: Fixed it via cargo-sources.json

File: xyz.safeworlds.hiit.yml (hunk 2 above)
Comment (mfep) [reply xyz.safeworlds.hiit.yml]: Exactly. Removed logging related env vars.

**Comment by hfiguiere** [comment]
bot, build xyz.safeworlds.hiit

**Pull request closed without merging**